packages/react-utils/src/components/AsyncSelect/ValueSetAsyncSelect/index.tsx

```diff
@@ -64,6 +65,21 @@ const getValueSetOptionsValue = (record: ValueSetContains) => {
   });
 };
 
+/**
+ * generate literal that represents the value to a dropdown select
+ *
+ * @param coding - coding to derive the literal value from
+ */
+export function getSelectValueForCoding(coding?: Coding) {
```

Comment:
Would be good to reuse this [here](https://github.com/onaio/fhir-web/blob/2580ede551bb93cec5d24dc4fafd1c47722a536e/packages/fhir-group-management/src/components/LocationInventory/utils.tsx#L76-L82).

Reply:
nice catch